zerver/lib/actions.py

```diff
@@ -5287,3 +5288,15 @@ def do_send_realm_reactivation_email(realm: Realm) -> None:
         'zerver/emails/realm_reactivation', realm,
         from_address=FromAddress.tokenized_no_reply_address(),
         from_name="Zulip Account Security", context=context)
+
+def get_zoom_video_call_url(realm: Realm) -> str:
+    response = request_zoom_video_call_url(
+        realm.zoom_user_id,
```

Comment:
Using the realm.zoom_user_id here creates all meetings for the user specified in the realm. In an organization this probably is the organization owner ?
Wouldn't it better to use the user_profile.email here and create the meeting for the "me" user ?
At least in our organization, it is so, that all Zulip emails are the same as the Zoom emails.
I did a patch for this, and it is working nice for us.
But I guess a more general soluation either would need to add a "zoom user id" to the userprofile, or at least respond with a comprehensive error message ?
Just a few thoughts to this.

Reply:
@marco @timabbott  - what do you think about these thoughts ?

We regularily run into the issue, that a zoom meeting already is in progress if another meeting should start, because of the meetings all created for the same Zoom user.

But if you would configure a Zoom admin user you could create the meetings for any other user, for example with the Zulip user email address or an optional future user-specific zoom email address ?

Reply:
@martin-huber This code was rewritten completely in Zulip Server 3.0 last year (#11874), and no longer associate a Zoom user with the realm. If you’re still using 2.2.x or earlier, we recommend you [upgrade](https://zulip.readthedocs.io/en/stable/production/upgrade-or-modify.html).

Reply:
oh, good, thanks for the hint ! We will give it a test asap !
I think, we did not yet update to 3.x because we also will need to update Ubuntu for that ... But let's see.
Thanks!